HistoricalDataFetcher.Classes/StartOptions/StartOptions.cs

```diff
@@ -18,5 +18,8 @@ public class StartOptions
         public string DBConnectionString { get; set; }
         [Option('f', "fqrs", HelpText = "The absolute path to the file containing the fullly qualified references")]
         public string FqrPath { get; set; }
+
+        [Option('i', "invalidcert", Default = false, HelpText = "Allow SSL certificate error in http response.")]
```

Comment:
this could use some better warning. Not sure I'd know what that means

Reply:
You can just write up an issue

Reply:
Issue created: https://github.com/metasys-server/historian/issues/2